pkgs/servers/waterfall/config.yml

```diff
@@ -0,0 +1,52 @@
+connection_throttle: 4000
+connection_throttle_limit: 3
+timeout: 30000
+stats: 69135692-e718-4282-a863-3231fb7fef22
```

Comment:
Better to let waterfall generate a unique value for everyone.
```suggestion
```

Reply:
Yes, so remove it from config?

Reply:
I changed it to 00000000-0000-0000-0000-000000000000